pkg/util/k8sutil/tpr_util.go

```diff
@@ -16,20 +16,24 @@ package k8sutil
 
 import (
 	"encoding/json"
-	"errors"
 	"fmt"
 	"net/http"
 	"time"
 
 	"github.com/coreos/etcd-operator/pkg/spec"
 	"github.com/coreos/etcd-operator/pkg/util/retryutil"
 
+	"github.com/Sirupsen/logrus"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/client-go/rest"
 )
 
 // TODO: replace this package with Operator client
 
+// ClusterTPRModifier is a modifier function to be used when atomically
+// updating a Cluster TPR.
+type ClusterTPRModifier func(*spec.Cluster)
```

Comment:
Modifier is usually an interface.
For a func, we should call it ModifyFunc or UpdateFUnc.